translations/fr-fr.yaml

```diff
@@ -0,0 +1,1280 @@
+      Lorsque vous aurez suivi quelques personnes et/ou groupes, nous changerons votre fil par défaut de "Global" à "Abonnements"'
+    action:
+      'Pourquoi ne pas écrire un post pour vous présenter, à l''aide du formulaire ci-dessous ?
+      Si vous êtes timide, un simple "Salut tout le monde, je suis nouveau sur Kitsu !" est suffisant,
+      mais si vous voulez montrer votre créativité et votre imagination, profitez !'
+  groups:
+    header: "Mes Groupes"
+  announcements:
+    button: "Voir Plus"
+# page: anime/manga
+media:
+  index:
+    search: "Rechercher par titre, personnage, doubleur…"
+    filter: "Filtrer"
+    remove: "Supprimer"
+    filters:
+      year: "Année"
+      score: "Note moyenne"
+      seasons: "Saisons"
+      streamers: "Diffuseurs (U.S.)"
+      units: "{type, select,
+        anime {Épisodes}
+        manga {Chapitres}
+        other {}}"
+      rating: "Classement"
+      categories: "Catégories"
+      types: "Types"
+    sort:
+      popularity: "Popularité"
+      rating: "Note Moyenne"
+      date: "Date"
+      recent: "Ajouté Récemment"
+  show:
+    navigation:
+      summary: "Résumé"
+      units: "{type, select,
+        anime {Épisodes}
+        manga {Chapitres}
+        other {}}"
+      characters: "Personnages"
+      reactions: "Réactions"
+      franchise: "Saga"
+    sidebar:
+      watch: "Regarder en streaming"
+    summary:
+      ratings:
+        community: "Approuvé à {rating}% par la communauté"
+        match: "Vous correspond à {rating}%"
+      ranking:
+        full:
+          popularity: "{type, select,
+            anime {Classé #{rank} (Animé le Plus Populaire)}
+            manga {Classé #{rank} (Manga le Plus Populaire)}
+            other {}}"
+          rating: "{type, select,
+            anime {Classé #{rank} (Animé le Mieux Noté)}
+            manga {Classé #{rank} (Manga le Mieux Noté)}
+            other {}}"
+        short:
+          popularity: "{type, select,
+            anime {#{rank} En Popularité}
+            manga {#{rank} En Popularité}
+            other {}}"
+          rating: "{type, select,
+            anime {#{rank} En Note}
+            manga {#{rank} En Note}
+            other {}}"
+      categories: "Catégories"
+      relationships:
+        header: "Plus de cette série"
+        more: "Voir tout de {title}"
+      reactions:
+        header: "Réactions"
+        more: "Voir plus de Réactions"
+        sorts:
+          -upVotesCount: "Populaire"
+          -createdAt: "Récent"
+        new: "Nouvelle Réaction"
+        edit: "Ma Réaction"
+        empty: "On dirait que personne n'a encore réagi (T_T)"
+      characters:
+        header: "Personnages"
+        more: "Voir tous les personnages"
+      titles:
+        header: "Titres Alternatifs"
+        en: "Anglais"
+        en_us: "Anglais (US)"
+        en_jp: "Romanisé"
+        ja_jp: "Japonais"
+        en_cn: "Pinyin"
+        zh_cn: "Chinois"
```

Comment:
Chinese has been changed to Simplified Chinese.

`zh_tw` key has been added for Traditional Chinese